maint.php

```diff
@@ -1137,28 +1361,43 @@ function clearFilter() {
 		strURL = 'maint.php?tab=webseer&action=edit&id=<?php print get_request_var('id');?>&clear=true&header=false';
 		loadPageNoHeader(strURL);
 	}
+
+	$(function() {
+		$('#rows, #associated').change(function() {
+			applyFilter();
+		});
```

Comment:
If you have add new filter fields, these should also be in the filter list.

Reply:
Are the new fields being picked up for filtering now?

Reply:
I must have missed submitting this response.  I think that those lines are for the webseer tab/function.  I did not add any filtering to webseer.  The filter fields for the devices tab is line 861.  The dynamic filtering seems to work.